### PR TITLE
Added RegistrationController.java and implemented adding

### PR DESCRIPTION
**Pull request by kosenkoNick**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,10 +22,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-security</artifactId>
-        </dependency>
+        <!--        <dependency>-->
```
Comment (VKoniukh) [inline pom.xml:25]: Should it be commented?

File: src/main/java/com/itacademy/cms/controller/CourseController.java [hunk 2]
```diff
@@ -51,11 +48,11 @@ public CourseGetDto showCourseById(@PathVariable("id") Long id) {
     return mapStructMapper.courseToCourseGetDto(courseService.getCourseById(id));
   }
 
-  @PostMapping
-  public void addNewCourse(@AuthenticationPrincipal User user,
-                           @RequestBody CoursePostDto coursePostDto) {
-    courseService.addCourse(coursePostDto, user);
-  }
+//  @PostMapping
```
Comment (VKoniukh) [inline src/main/java/com/itacademy/cms/controller/CourseController.java:51]: this

File: src/main/java/com/itacademy/cms/controller/RegistrationController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  @PostMapping("/users/register")
+  public String collectData(@Valid @ModelAttribute("user") UserDto user,
+                            BindingResult bindingResult) {
+    if (bindingResult.hasErrors()) {
+      return "...";
+    }
+
+//    test
+//    user.setFirstName("nick");
```
Comment (VKoniukh) [inline src/main/java/com/itacademy/cms/controller/RegistrationController.java:34]: and this

**Review by VKoniukh — APPROVED** [review]

**Comment by dmakhun** [comment]
any reason test classes are deleted?

**Pull request merged**